public/index.html

```diff
@@ -0,0 +1,12 @@
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Document</title>
+<link rel="stylesheet" href="style.css">
+<script src="https://cdn.tailwindcss.com"></script>
+</head>
```

Comment:
This `head` tag doesn't have a opening pair

Reply:
yes all corrected.